rasa/core/policies/intent_ted_policy.py

```diff
@@ -397,26 +403,70 @@ def run_training(
         self.compute_label_quantiles_post_training(model_data, label_ids)
 
     def _collect_action_metadata(
-        self, domain: Domain, similarities: np.array
-    ) -> Dict[Text, Dict[Text, float]]:
+        self, domain: Domain, similarities: np.array, query_intent: Text
+    ) -> Dict[Text, Any]:
         """Adds any metadata to be attached to the predicted action.
 
-        Similarities for all intents and their thresholds are attached as metadata.
+        Metadata schema looks like this:
+
+        {
+            "query_intent": <metadata of intent that was queried>,
+            "ranking": <sorted list of metadata corresponding to all intents.
+                        Sorting is based on predicted similarities.>
+        }
+
+        Each metadata dictionary looks like this:
```

Comment:
"Each metadata dictionary in the ranking field looks like this:"

At first it wasn't clear to me how the top schema related to the bottom.

Reply:
Oh actually I see that both fields contain metadata, where query_intent is a singleton and ranking is a list. Perhaps maybe worth noting that the query_intent metadata is also in ranking?

Reply:
> Perhaps maybe worth noting that the query_intent metadata is also in ranking?

Added that in.